wfe-core/src/main/java/ru/runa/wfe/commons/Utils.java

```diff
@@ -223,6 +223,16 @@ public static String getReceiveMessageNodeSelector(VariableProvider variableProv
         Collections.sort(selectors);
         return Joiner.on(MESSAGE_SELECTOR_DELIMITER).join(selectors);
     }
+    
+    public static Map<String, String> getVariableNameMappedNamePair(BaseMessageNode messageNode, String variableName) {
```

Comment:
Move function to BaseMessageNode and change to boolean areSelectorsContainVariable(String) because variable could no t be defined twice at selectors (in normal case).